src/reducers/comments.tsx

```diff
@@ -621,13 +642,16 @@ const reducer: Reducer<CommentsState, ActionType<typeof actions>> = (
 
         const { key, info } = getKeyAndInfo({
           byKey,
+          commentId: undefined,
```

Comment:
This code is a little confusing because it's looping through a list of comments yet setting `commentId` to `undefined`. It looks like a mistake. I think this comment will help:

```suggestion
          // Since we'll be appending to a collection of *all* comments,
          // the commentId must always be undefined.
          commentId: undefined,
```